corehq/apps/reports/util.py

```diff
@@ -453,12 +453,14 @@ def add_tableau_user(domain, username):
     these details to the Tableau instance.
     '''
     session = TableauAPISession.create_session_for_domain(domain)
-    user = TableauUser.objects.create(
+    user, created = TableauUser.objects.get_or_create(
         server=session.tableau_connected_app.server,
         username=username,
         role='Viewer',
     )
-    new_id = session.create_user(username, 'Viewer')
+    if not created:
+        return
+    new_id = session.create_user('HQ/' + username, 'Viewer')
```

Comment:
could be nice to have a small utility function that formats the username just so that it's clear across multiple places where it happens